development/acpica/acpica.SlackBuild

```diff
@@ -74,7 +74,7 @@ find -L . \
  -o -perm 440 -o -perm 400 \) -exec chmod 644 {} \;
 
 cd generate/unix
-  CFLAGS="$SLKCFLAGS -fcommon -Wno-error=format-overflow -Wno-error=stringop-truncation" \
+  CFLAGS="$SLKCFLAGS" \
   CXXFLAGS="$SLKCFLAGS" \
```

Comment:
instead of CFLAGS and CXXFLAGS it should use OPT_CFLAGS now

Reply:
Done, thank you